modules/viewing-the-details-of-a-pipeline-version.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+
+[id="viewing-the-details-of-a-pipeline-version_{context}"]
+= Viewing the details of a pipeline version
```

Comment:
See previous comment RE: potentially combining this with modules/viewing-pipeline-versions.adoc.

Reply:
In general, we have used separate modules for things like this as there are different screens involved, each screen showing different components.